internal/service/resourcepolicy/data_source_plural.go

```diff
@@ -0,0 +1,56 @@
+package resourcepolicy
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
+)
+
+var _ datasource.DataSource = &resourcePolicysDS{}
+var _ datasource.DataSourceWithConfigure = &resourcePolicysDS{}
+
+const (
+	dataSourceNamePlural = "resource_policies"
+	errorReadDSP         = "error reading plural data source " + dataSourceNamePlural
```

Comment:
knit: mongodb_ is missing in the name? 

i used `fullResourceName` like in:  https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/service/mongodbemployeeaccessgrant/resource.go#L16

Reply:
Thanks. Updated in c2ee9efa28658d86eaff5155f0ef5b0f05f1b1af